### PR TITLE
soft reset

### PR DESCRIPTION
**Pull request by FrostyMoS**
I guess because of the repo reset, i needed to soft reset this branch, resulting in a single commit

### CODE REVIEW

File: assets/styleable/guns/shotgun.svg
Comment (CarnVanBeck) [inline assets/styleable/guns/shotgun.svg:1]: looks like the smg file with a border

File: css/dices.css
Comment (CarnVanBeck) [inline css/dices.css:1]: just dice as the file name, it's already the plural

File: scripts/gunCard.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+export class GunCardSheet extends ItemSheet {
+    
+    get template() {
+
```
Comment (CarnVanBeck) [inline scripts/gunCard.js]: maybe something like

let sheet = gunCard.html
if (!game.user.isGM && this.actor.limited) {
    sheet = limited-sheet.html
}
return \`systems/vtt-bunkersandbadasses/template/${sheet}\`;

File: scripts/gunCard.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        const path = "systems/vtt-bunkersandbadasses/templates/gunCard.html";
+        if (!game.user.isGM && this.actor.limited) {
+			return path + "limited-sheet.html";
+		}
+        return "systems/vtt-bunkersandbadasses/templates/gunCard.html";
```
Comment (CarnVanBeck) [inline scripts/gunCard.js]: after merging the start of the path "systems/vtt-bunkersandbadasses" can be replaced by BADASS.systemPath

File: scripts/gunCard.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+		// Active Effect management
+		html.on('click', '.effect-control', (ev) =>
+			onManageActiveEffect(ev, this.item)
+		);
+	}
+		
+}
+
+Items.registerSheet("bunkers-and-badasses", GunCardSheet, {
```
Comment (CarnVanBeck) [inline scripts/gunCard.js]: I think this makes more sense in the badass.mjs

File: scripts/gunCard.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    types: ["gun"],
+    makeDefault: false
+});
+
+Hooks.on('ready', () => {
```
Comment (CarnVanBeck) [inline scripts/gunCard.js]: I think this whole Hook can be removed for now

File: scripts/handlebars.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+		// elements
+		elementAcid: "systems/vtt-bunkersandbadasses/assets/styleable/elements/acid.html",
+		elementCryo: "systems/vtt-bunkersandbadasses/assets/styleable/elements/cryo.html",
+	};
+
+	// Load the template parts
+	return loadTemplates(templatePaths);
+};
+
+Hooks.once( "init", function() {
```
Comment (CarnVanBeck) [inline scripts/handlebars.js]: Should be in badass.mjs, so that there is only one init hook in the system

File: system.json [hunk 6]
```diff
@@ -21,8 +21,17 @@
         "minimum": 12,
         "verified": "12.331"
     },
-    "esmodules": ["badass.mjs"],
-    "styles": ["css/badass.css"],
+    "esmodules": [
+        "badass.mjs",
+        "scripts/gunCard.js",
```
Comment (CarnVanBeck) [inline system.json]: gunCard.js can be removed if the registerSheet is moved to badass.mjs

File: system.json [hunk 7]
```diff
@@ -21,8 +21,17 @@
...
+        "scripts/handlebars.js"
```
Comment (CarnVanBeck) [inline system.json]: handlebards.js can be removed if the Hook is moved to badass.mjs

**Review by CarnVanBeck — CHANGES_REQUESTED** [review]
Only 2 things that I would change before merging, gun.js registerSheet and handlebars.js Hook

Rest are just some comments that don't need to be addressed right now.

File: assets/styleable/guns/shotgun.svg
Comment (FrostyMoS) [reply assets/styleable/guns/shotgun.svg:1]: skipped this file in the creation process and didn't noticed its not done yet

File: scripts/gunCard.js (hunk 1 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: This was a template from a module and wont be necessary in the future

File: scripts/gunCard.js (hunk 2 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: So it could be done before merging ?

File: scripts/gunCard.js (hunk 3 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: badass.mjs came with a later pull from main, but the question is. Should the call for this be in badass.mjs or in an imported inventoryItems.mjs ?

File: scripts/gunCard.js (hunk 4 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: Relic from the Template, will be removed soon

File: css/dices.css
Comment (FrostyMoS) [reply css/dices.css:1]: Renamed

File: scripts/gunCard.js (hunk 1 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: deleted, because its nor necessary for now

File: scripts/gunCard.js (hunk 2 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: TODO

File: scripts/gunCard.js (hunk 3 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: moved to badass.mjs

File: scripts/gunCard.js (hunk 4 above)
Comment (FrostyMoS) [reply scripts/gunCard.js]: Removed

File: scripts/handlebars.js (hunk 5 above)
Comment (FrostyMoS) [reply scripts/handlebars.js]: Moved to badass.mjs

File: system.json (hunk 6 above)
Comment (FrostyMoS) [reply system.json]: removed/Moved to badass.mjs

File: system.json (hunk 7 above)
Comment (FrostyMoS) [reply system.json]: Moved to badass.mjs

**Review by CarnVanBeck — APPROVED** [review]

**Pull request merged**